Cargo.toml

```diff
@@ -1,21 +1,21 @@
 [package]
-name = "vault_client"
-version = "0.3.0"
 authors = ["Metaswitch Networks Ltd"]
-
-description = "A client library for HashiCorp Vault"
-readme = "README.md"
-keywords = ["vault", "tls"]
 categories = ["authentication"]
-
+description = "A client library for HashiCorp Vault"
+keywords = [
+    "vault",
+    "tls",
+]
 license = "Apache-2.0/MIT"
+name = "vault_client"
+readme = "README.md"
+version = "0.3.0"
```

Comment:
Are you using an old version of `cargo-edit`? That tends to sort the package metadata like this.

`cargo install cargo-edit --vers 0.3.0-beta.1`